crates/router/tests/connectors/adyen_uk_ui.rs

```diff
@@ -727,4 +744,10 @@ fn should_make_adyen_dana_payment_test() {
     tester!(should_make_adyen_dana_payment);
 }
 
+#[test]
+#[serial]
+fn should_make_adyen_online_banking_fpx_payment_test() {
+    tester!(should_make_adyen_online_banking_fpx_payment);
+}
+
 // https://hs-payments-test.netlify.app/paypal-redirect?amount=70.00&country=US&currency=USD&mandate_data[customer_acceptance][acceptance_type]=offline&mandate_data[customer_acceptance][accepted_at]=1963-05-03T04:07:52.723Z&mandate_data[customer_acceptance][online][ip_address]=127.0.0.1&mandate_data[customer_acceptance][online][user_agent]=amet%20irure%20esse&mandate_data[mandate_type][multi_use][amount]=700&mandate_data[mandate_type][multi_use][currency]=USD&apikey=dev_uFpxA0r6jjbVaxHSY3X0BZLL3erDUzvg3i51abwB1Bknu3fdiPxw475DQgnByn1z
```

Comment:
Why is this comment present here? it also mentions a API_KEY

Reply:
@srujanchikke will pick this up in a separate PR